src/packaging/dmlinux/Makefile

```diff
@@ -36,10 +36,10 @@ prepare: prepare.out
 prepare.out: FORCE
 	$(if $(DMLINUX_SOURCE_DIR),,$(error Must set DMLINUX_SOURCE_DIR))
 	rm -rf $(WORK_DIR)
-	mkdir -p $(WORK_DIR)/$(PACKAGE)/dm-vdo
+	mkdir -p $(WORK_DIR)/$(PACKAGE)
 	sed -e "s/@VERSION@/$(VDO_VERSION)/g" kvdo.spec >$(WORK_DIR)/kvdo.spec
 	cp kernel/Makefile $(WORK_DIR)/$(PACKAGE)
-	cp $(DMLINUX_SOURCE_DIR)/drivers/md/dm-vdo/*.[ch] $(WORK_DIR)/$(PACKAGE)/dm-vdo > prepare.out
+	cp -r $(DMLINUX_SOURCE_DIR)/drivers/md/dm-vdo $(WORK_DIR)/$(PACKAGE)/dm-vdo > prepare.out
```

Comment:
Is there a reason we're not just copying source files anymore? I'm worried about copying object files and things if the kernel source happened to have been built before running this target.

Reply:
It's a newly-cloned repo when CI runs the tests, right? 

I just thought this was easier that having to copy each level of the directory tree explicitly. Perhaps there's a way to just copy the .[ch] files, though.

Reply:
I've replaced the cp command with a slightly more sophisticated find command that copies just the *.c and *.h files in the whole directory structure.